tests/storage-vm

```diff
@@ -612,7 +612,7 @@ for poolDriver in $poolDriverList; do
         lxc config set storage.images_volume "${poolName}"/images
         lxc publish v1 --alias v1image
         lxc delete v1
-        lxc launch v1image v2 -s "${poolName}"
+        lxc launch v1image v2 --vm -s "${poolName}"
```

Comment:
That's the only place where a VM image alias was used to launch an instance without providing that `--vm` flag.

From @tomponline's comment, I understand we don't want to break this now but maybe we should be more explicit in our testing. How about adding a test for that to `tests/vm`?

Reply:
I'm fine with a comment but I'll submit a PR to add an explicit test for the implicit `--vm` flag ;)

Reply:
@kadinsayani could you please leave the `--vm` flag out but put a comment as requested by Tom. Thanks